utils/docker/Dockerfile.centos.8

```diff
@@ -6,7 +6,7 @@
 #
 
 # Pull base image
-ARG BASE_DISTRO=centos:8
+ARG BASE_DISTRO=rockylinux/rockylinux:8
```

Comment:
Should we rename this file accordingly to `Dockerfile.rocky.8`?

Reply:
Not on the 2.0 branch, a lot of this code and in particular the github actions work has been completely re-written to this is not the place to make that change.

Reply:
I'm not sure how any of that is related to simply renaming the Dockerfile (and it's reference I suppose) to reflect what it is actually is building with.  In any case, it was just a suggestion so I won't belabour it.

It does seem troubling though that we are removing 1 of our 2 supported platforms from an entire pipeline of testing though.  Maybe that is fine for 2.0 given that we also build and test in Jenkins.